zio-http-gen/src/test/resources/ComponentAliasOrderId.scala

```diff
@@ -0,0 +1,9 @@
+package test.component
+
+import zio.prelude.Newtype
+import zio.schema.Schema
+import java.util.UUID
+
+object OrderId extends Newtype[UUID] {
```

Comment:
You need to add this when you declare a newtype
```suggestion
type OrderId = OrderId.Type
object OrderId extends Newtype[UUID] {
```

Reply:
@jivanic-demystdata thanks for taking the time to look at this :)

scala 2 does not support top level type declarations.
Generated code needs to be valid scala 2 as well.

Regardless, the Newtype aliases has already been implemented in #3002 

This PR isn't a direct extension, but it needs to work with the alias flag as well.